javascript/package.json

```diff
@@ -28,18 +28,12 @@
   },
   "license": "MIT",
   "author": "Nathan Hopkins <natehop@gmail.com>",
-  "source": "./stimulus_reflex.js",
-  "main": "./dist/stimulus_reflex.js",
-  "module": "./dist/stimulus_reflex.module.js",
-  "esmodule": "./dist/stimulus_reflex.modern.js",
+  "main": "./stimulus_reflex.js",
+  "module": "./stimulus_reflex.js",
   "scripts": {
-    "prepare": "yarn build",
-    "postinstall": "node ./scripts/post_install.js",
```

Comment:
Why did we remove the `postinstall` script here?

Reply:
good catch. fixed on master.